### ISSUE TITLE
Cursor not reset if changed by hover effect when widget disappears

### ISSUE THREAD

**Issue by totycro** [unknown-horizons/unknown-horizons#1747]
c.f. http://simplest-image-hosting.net/png-0-2012-05-13-15-53-10-465827

or send a chat message via enter.

The cursor is currently not notified of the widget being hidden. vdaras is working on a possibility to catch this, so it's blocked by his work. It will have to be used in make_cursor_change_on_hover_class in pychan_util.py

[> Link to originally reported Trac ticket <] (http://trac.unknown-horizons.org/t/ticket/1747)

**Comment by squiddy** [comment]
Closing in favor of #2416
